group.go

```diff
@@ -86,5 +93,9 @@ func (g *TaskGroupWithContext) Wait() error {
 	case <-g.ctx.Done():
 	}
```

Comment:
Oh, good catch @NOMORECOFFEE!. If you don't mind, please open a PR with suggested changes and I can review it later. Otherwise I can send one. Thank you! 🙌

Reply:
I just released [v1.8.3](https://github.com/alitto/pond/releases/tag/v1.8.3) with a fix for this, please check it out and let me know if you find anything. Thank you!